cdap-client-tests/src/main/java/co/cask/cdap/client/app/AppReturnsArgs.java

```diff
@@ -49,7 +50,8 @@ public void configure() {
     @Path(ENDPOINT)
     @GET
     public void handler(HttpServiceRequest request, HttpServiceResponder responder) {
-      responder.sendJson(200, getContext().getRuntimeArguments());
+      TreeMap<String, String> values = new TreeMap<>(getContext().getRuntimeArguments());
```

Comment:
@gsps1 The reason for the failure was the comparison was being earlier done on Json strings instead of the Maps. If the order of the keys happened to be the same then the tests used to pass. Put once in a while the order ended up being different and the test failed. So in order to increase the changes of failure, I had added the logic to sort the keys. But removed it after @chtyim comment.